packages/nx/src/utils/nx-cloud-utils.ts

```diff
@@ -1,13 +1,27 @@
 import { NxJsonConfiguration, readNxJson } from '../config/nx-json';
 
-export function isNxCloudUsed(nxJson: NxJsonConfiguration) {
-  return (
+// Returns tuple [isCloudUsed, cloudConfigurationName]
+export function isNxCloudUsed(nxJson: NxJsonConfiguration): [boolean, string] {
```

Comment:
I think this function should only return `boolean`.

1. It's weird otherwise
2. There's no real magic going on here which can't be moved into `getCloudOptions` or somewhere else.